test/entrypoint.sh

```diff
@@ -14,17 +14,16 @@ wait_tcp_port() {
     local host="$1" port="$2"
 
     # see http://tldp.org/LDP/abs/html/devref1.html for description of this syntax.
-    while ! exec 6<>/dev/tcp/$host/$port; do
-	echo "$(date) - still trying to connect to $host:$port"
-	sleep 1
+    for n in `seq 1 30` ; do if exec 6<>/dev/tcp/$host/$port; then
```

Comment:
I think this would be more readable with the `if exec 6<> ...` on the proceeding line